plugins/lsp/default.nix

```diff
@@ -69,7 +69,6 @@ in
         "lua-ls"
         "marksman"
         "metals"
-        "nil_ls"
```

Comment:
This removal notice is overdue for removal anyway (2024-03-14); should we remove the whole thing instead of just the `nil_ls` case?

Or maybe it's better to wait until after 24.05 before removing any warnings/assertions?

Reply:
Removing for now, let me know if you'd prefer a different approach